webapp/channels/src/components/multiselect/multiselect.tsx

```diff
@@ -32,9 +33,9 @@ export type Props<T extends Value> = {
     ariaLabelRenderer: getOptionValue<T>;
     backButtonClick?: () => void;
     backButtonClass?: string;
-    backButtonText?: string;
-    buttonSubmitLoadingText?: ReactNode;
-    buttonSubmitText?: ReactNode;
+    backButtonText?: string | MessageDescriptor;
+    buttonSubmitLoadingText?: ReactNode | MessageDescriptor;
```

Comment:
Maybe we should specify which one we want for these. Not sure how much more refactoring that would entail, but it would be less confusing.

Reply:
I did try to do that in some places, but it really required a lot of extra changes that were more involved, so I went with more permissive ones for now. We can probably revisit it in the future, but it was out of scope for what I wanted to do right now